agentverse/llms/openai.py

```diff
@@ -86,15 +92,15 @@ class OpenAIChatArgs(BaseModelArgs):
 #         )
 
 
-@llm_registry.register("gpt-3.5-turbo")
+@llm_registry.register("gpt-35-turbo")
```

Comment:
I think removing the original "gpt-3.5-turbo" might break compatibility with the official OpenAI API. It would be better to keep the original line and simply add this new one